### PR TITLE
Basename: Fix mobile badge and navigation layout

### PR DESCRIPTION
**Pull request by kirkas**
#### Before

image

#### After

image

#### Before

image

#### After

image

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/web/src/components/Basenames/UsernameProfileSectionBadges/Badges/index.tsx [hunk 1]
```diff
@@ -153,7 +153,7 @@ export function BadgeImage({ badge, claimed, score, size, name }: BadgeImageProp
   const showTalentScore = Boolean(claimed && score && badge === 'TALENT_SCORE');
 
   return (
-    <div className="group relative flex h-[160px] w-[160px] items-center justify-center">
```
Comment (JFrankfurt) [inline apps/web/src/components/Basenames/UsernameProfileSectionBadges/Badges/index.tsx:156]: I would expect removal of these values to have a significant effect on layout. Double checking this has been tested on big viewports too?

File: apps/web/src/components/Basenames/UsernameProfileSectionBadges/index.tsx [hunk 2]
```diff
@@ -19,14 +18,14 @@ function BadgesLoop({
   currentWalletIsOwner?: boolean;
 }) {
   return (
-    <ul className="mb-12 mt-6 flex flex-col flex-wrap items-center gap-8 sm:flex-row sm:items-start">
+    <ul className="mt-6 grid grid-cols-2 gap-4 md:flex md:flex-row md:flex-wrap md:items-center md:gap-8">
```
Comment (JFrankfurt) [inline apps/web/src/components/Basenames/UsernameProfileSectionBadges/index.tsx:21]: I've found sm to be larger than most (all?) the mobile viewports I've worked with. Just curious what you found the change to md got you?

**Review by JFrankfurt — APPROVED** [review]

File: apps/web/src/components/Basenames/UsernameProfileSectionBadges/index.tsx (hunk 2 above)
Comment (kirkas) [reply apps/web/src/components/Basenames/UsernameProfileSectionBadges/index.tsx:21]: we used md in every other sections/pages for basename, that seemed to fit the design correctly here (tested on Desktop/Mobile)

File: apps/web/src/components/Basenames/UsernameProfileSectionBadges/Badges/index.tsx (hunk 1 above)
Comment (kirkas) [reply apps/web/src/components/Basenames/UsernameProfileSectionBadges/Badges/index.tsx:156]: yes desktop & mobile

**Pull request merged**